platforms/ide/tooling-api/src/main/java/org/gradle/tooling/internal/protocol/problem/InternalBasicProblemDetails2.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.tooling.internal.protocol.problem;
+
+import org.gradle.tooling.internal.protocol.InternalFailure;
+
+import javax.annotation.Nullable;
+
+public interface InternalBasicProblemDetails2 extends InternalBasicProblemDetails {
```

Comment:
❓ The convention in the protocol package is to use the `VersionX` suffix for classes, i.e. this type probably should be named `InternalBasicProblemDetailsVersion2`.